CDL-Dryad/stash
# Payment logic framework

Adding the framework for payment logic. Adding an explicit check for institutional coverage of the Data Processing Charge.

To test:
- In a tenant file, set `covers_dpc: true`. 
- When a submission is made with this tenant active, the "Payment" paragraph will not appear on the "Review and Submit" page.
- When an item is published (or embargoed) with this tenant active, a Stripe invoice will not be generated.

Journal payments and fee waivers are not covered in this PR; they default to `false`.
